.circleci/config.yml

```diff
@@ -35,40 +50,46 @@ jobs:
                 name: Run eslint
                 command: ./node_modules/.bin/eslint src
                 when: always
+            
+            - run:
+                name: Run prettier
+                command: |
+                    ./node_modules/.bin/prettier --config .prettierrc --write src/**/*.js src/**/**/*.js
```

Comment:
```suggestion
                    npm run lint:lint.prettier
```
🌴, also this is the `format` version, you want the `lint` version right? Anyway it's better if we can have CI run commands from package.json, so that we know it's doing the same thing a dev will do when they're writing their code.

That said those command names `lint:lint.prettier` and `lint:format.prettier` and such are awkward. `lint...` commands should just test, `format...` should edit the files. So you could just call these `lint.prettier` and `format.prettier`, or follow the convention we have in Dash (examples from DCC but all the pieces basically do the same):
- [`lint`](https://github.com/plotly/dash/blob/6eaf2e17c25f7ca1847c41aafeb18e87c586cb9f/components/dash-core-components/package.json#L32) runs all the linting subcommands, [`format`](https://github.com/plotly/dash/blob/6eaf2e17c25f7ca1847c41aafeb18e87c586cb9f/components/dash-core-components/package.json#L31) runs all the formatting subcommands
- The [subcommands](https://github.com/plotly/dash/blob/6eaf2e17c25f7ca1847c41aafeb18e87c586cb9f/components/dash-core-components/package.json#L15-L21) are all called `private::lint.<tool>` and `private::format.<tool>`